clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -5,29 +5,78 @@ import {
   PrivacyExperience,
   ConsentMechanism,
   PrivacyNotice,
+  ExperienceConfig,
 } from "../lib/consent-types";
 
-type NoticeKeys = Array<PrivacyNotice["notice_key"]>;
-
 const ConsentButtons = ({
```

Comment:
there are now 3 components in this file:

* `ConsentButtons`
* `NoticeConsentButtons`
* `TcfConsentButtons`

The original `ConsentButtons` had a lot of notice specific logic in it, so a refactoring as needed!